databroker/src/broker.rs

```diff
@@ -1627,7 +1628,7 @@ impl<'a, 'b> AuthorizedAccess<'a, 'b> {
             return Err(SubscriptionError::InvalidInput);
         }
 
-        let (sender, receiver) = mpsc::channel(10);
+        let (sender, receiver) = broadcast::channel(100);
```

Comment:
"You're right that it's probably "overkill" to use one broadcast channel per subscriber, but the other nice semantic of the broadcast channel is that it will drop the oldest value when the subscriber is lagging." (John Argerus)

Have you put a thought into this?

Reply:
Since we're moving to a model of dropping notifications when a subscriber lags, I think the buffer size should be configurable by the subscriber (i.e. the caller of `pub async fn subscribe(...)`), instead of hard coding it.

I think it's most likely that subscribers are only interested in receiving the latest value without caring whether an intermediate value was dropped (i.e. `broadcast::channel(1)`). But since there _can_ be other use cases it should probably be configurable.

For the previous model, i.e. nothing was to be dropped, the buffer acted as a way to not immediately put back pressure on the provider.

Reply:
I suppose you mean only to add the size field parameter to `kuksa.val.v2` subscribe request, but what would be the appropriate default value to be used then for `kuksa.val.v1` and `sdv.databroker.v1`?

Reply:
I guess the default at the moment ought to be `1` as that would be the better choice if latency is what we prioritize.

Reply:
Done

Reply:
> I suppose you mean only to add the size field parameter to `kuksa.val.v2` subscribe request, but what would be the appropriate default value to be used then for `kuksa.val.v1` and `sdv.databroker.v1`?

I actually didn't mean that you must add it to any GRPC interface right now (in this PR). I just said that it should be configurable in the (internal) call to subscribe, i.e. `pub async fn subscribe(...)`.